docs/specs/bind-unbind-workflow.md

```diff
@@ -0,0 +1,22 @@
+## Bind / Unbind New Work flow
+
+The goal of this new work flow is to give more control over the actions that the database takes on bind and unbind to the APB author. This means that we want the apb to be launched on on bind and unbind. 
+
+### Default Settings Needed
+* set `launch_apb_on_bind: true` - If this is set to false we will behave the exact same as today, handing back the credentials received at provision to the service catalog.
+
+
+### Brokers Responsibility
+* The broker will save the credentials for each bind. this will be saved in etcd under the `extracted_credentials` directory. The key will be the binding id.
+* The broker will be responsible for providing the provisioned credentials to the APB as extra vars during bind and unbind. Example
+```
```

Comment:
I tried to create examples that would show what was needed. Let me know if that addressed your question of if it still needs clarification.